frontend/src/pages/project/Deployments.vue

```diff
@@ -1,94 +1,151 @@
 <template>
```

Comment:
View this file with [?w=1](https://github.com/flowforge/flowforge/pull/1204/files?w=1) for easier review